Added owners, connections tables, added owner, notes fields to asset,…

ejaxon opened the pull request:
1. Added a _connections_ table with name, description, and type (as an ENUM). Did **not** add the actual connections to the table or remove _loctype_ from location information in assets
2. Added an _owners_ table
3. Added two columns to the assets table: _owner_ (integer id that references the new owners table) and _notes_ (text field).

ejaxon commented:
Obviously we'll need to rebuild the DB.

bitslayer42 reviewed (approved):
I think I should be able to go ahead and add the fields to the production db. Does that sound right?

ejaxon commented:
Yes, I think so.

BTW I added 2 more changes: added change log notes to two assets (https://github.com/cityofasheville/bedrock2/blob/update-tables/src/db/bedrock-db-data/test_data/assets/mwbe_vendors_by_commodity.mun/mwbe_vendors_by_commodity.mun.json and https://github.com/cityofasheville/bedrock2/tree/update-tables/src/db/bedrock-db-data/test_data/assets/mwbe_vendors_by_commodity_non_aggregated.mun).

LMK if there's something better than using the double-backslashed newline to create line breaks.
